### PR TITLE
AST-1596 - Match up WP block editor & their respective frontend UI

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Filter to use to test new UI
```
add_filter( 'astra_get_option_enable-brand-new-editor-experience', '__return_true' );
```
- Sidebar setting's controller update from dropdown to image picker - https://share.bsf.io/JruGKQNm - https://share.bsf.io/Z4u1WEJd
- "Disable Page Title" toggle setting removed and managed easily near title input - https://share.bsf.io/NQuxy0Ek
- All old CSS removed we added tilldate (except required ones like easy interactive hover box-shadow, spacer BG etc) 
- Users can override sizes by following ways -
- 1. Either by CSS (overriding by vars)
- 2. by filters provided
```
// But this will only update values on frontend not in editor mode.
:root body {
    --ast-content-width-size: 910px;
    --ast-wide-width-size: 1200px;
}
```
- Filters to update sizes - 
```
function astra_update_block_content_size( $content_size ) {
	return '840px'; // Default 910px overriden by 840px.
}

function astra_update_block_wide_size( $wide_size ) {
	return '1000px'; // Default 1200px overriden by 1000px.
}

add_filter( 'astra_block_content_width', 'astra_update_block_content_size' );
add_filter( 'astra_block_wide_width', 'astra_update_block_wide_size' );
```

### Types of changes
- New feature (non-breaking change )

### How has this been tested?
- Check as per plan & card comments - https://wp-astra.atlassian.net/browse/AST-1596
- https://brainstormforce.slack.com/archives/C0FB28KK3/p1645688114969249
- Test every block of WP
- Check block layouts
- Check site & Astra layouts
- Check by importing layout if anything breaking

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -49,11 +49,59 @@
...
+    "typography": {
+			"customFontSize": true,
+      "customLineHeight": true,
+      "dropCap": true,
+      "fontSizes": [
+				{
+					"size": "14px",
+          "name": "Tiny",
+					"slug": "tiny"
+				},
```
Comment (premanshup) [inline theme.json:69]: @imnavanath Please add proper indentation. Please check if we can implement JSON linting for theme.json file. 
[https://github.com/brandonramirez/grunt-jsonlint](https://github.com/brandonramirez/grunt-jsonlint) 
Maybe we can use the above grunt linter.

Comment (imnavanath) [reply theme.json]: Resolved 👍

Comment (imnavanath) [reply theme.json]: Included package for this.

File: functions.php [hunk 2]
```diff
@@ -20,6 +20,10 @@
 define( 'ASTRA_THEME_DIR', trailingslashit( get_template_directory() ) );
 define( 'ASTRA_THEME_URI', trailingslashit( esc_url( get_template_directory_uri() ) ) );
 
+/**
+ * Added this filter for easy testing package & branch, will remove later.
+ */
+add_filter( 'astra_get_option_enable-v3-8-0-ui', '__return_true' );
```
Comment (premanshup) [inline functions.php:26]:
```suggestion
```

Comment (imnavanath) [reply functions.php]: Removed

**Review by SimranChougule1 — APPROVED** [review]

**Pull request merged**